include/openthread/ip6.h

```diff
@@ -105,6 +117,7 @@ typedef struct otNetifAddress
 {
     otIp6Address           mAddress;                ///< The IPv6 unicast address.
     uint8_t                mPrefixLength;           ///< The Prefix length (in bits).
+    uint8_t                mAddressOrigin: 2;       ///< The IPv6 address origin.
```

Comment:
Needed to pass pretty check:
```suggestion
    uint8_t                mAddressOrigin : 2;      ///< The IPv6 address origin.
```